src/pages/HelpPage.js

```diff
@@ -129,6 +129,11 @@ function HelpPage() {
           pressing the <code>;</code> key. This also changes the keyboard
           shortcuts, which may be more convenient to use.
         </Typography>
+        <Typography variant="body1" gutterBottom>
```

Comment:
Maybe this is a stretch -- but it would be really cool if this section of the help page (lines 120-130) updated the text displayed based on your current keyboard layout!

Reply:
Done, wasn't all that difficult using contexts.